### ISSUE TITLE
update pachctl put file docs' examples to new syntax

### ISSUE THREAD

**Issue by gabrielgrant** [pachyderm/pachyderm#3814]
The `put file` usage has been updated, but all subsequent examples don't use the `<repo>@<branch-or-commit>[:<path/in/pfs>]` syntax (thought they do have `put file` rather than `put-file`, meaning they'll all fail as-written)

also, the `-c` example should probably be removed

```
$ pachctl put file --help
Put a file into the filesystem.  This supports a number of ways to insert data into pfs.

Usage:
  pachctl put file <repo>@<branch-or-commit>[:<path/in/pfs>] [flags]

Examples:

# Put data from stdin as repo/branch/path:
$ echo "data" | pachctl put file repo branch path

# Put data from stdin as repo/branch/path and start / finish a new commit on the branch.
$ echo "data" | pachctl put file -c repo branch path

# Put a file from the local filesystem as repo/branch/path:
$ pachctl put file repo branch path -f file

# Put a file from the local filesystem as repo/branch/file:
$ pachctl put file repo branch -f file

# Put the contents of a directory as repo/branch/path/dir/file:
$ pachctl put file -r repo branch path -f dir

# Put the contents of a directory as repo/branch/dir/file:
$ pachctl put file -r repo branch -f dir

# Put the contents of a directory as repo/branch/file, i.e. put files at the top level:
$ pachctl put file -r repo branch / -f dir

# Put the data from a URL as repo/branch/path:
$ pachctl put file repo branch path -f http://host/path

# Put the data from a URL as repo/branch/path:
$ pachctl put file repo branch -f http://host/path

# Put the data from an S3 bucket as repo/branch/s3_object:
$ pachctl put file repo branch -r -f s3://my_bucket

# Put several files or URLs that are listed in file.
# Files and URLs should be newline delimited.
$ pachctl put file repo branch -i file

# Put several files or URLs that are listed at URL.
# NOTE this URL can reference local files, so it could cause you to put sensitive
# files into your Pachyderm cluster.
$ pachctl put file repo branch -i http://host/path
...
```

**Comment by Tryneus** [comment]
@msteffen fixed this in #3809 - sorry I didn't catch it sooner!

**Comment by gabrielgrant** [comment]
ah, sweet, two steps ahead!
